src/interface/keyboard_shortcut.c

```diff
@@ -500,6 +500,21 @@ static void shortcut_open_chat_window()
 	chat_toggle();
 }
 
+static void shortcut_quick_save_game()
+{
+	log_error("quicksave");
+	// Do a quick save in playing mode and a regular save in Scenario Editor mode. In other cases, don't do anything.
+	if (RCT2_GLOBAL(RCT2_ADDRESS_SCREEN_FLAGS, uint8) == SCREEN_FLAGS_PLAYING) {
+		log_error("quicksave2");
```

Comment:
You can't save in the Track Designer or the Track Designs manager.
There is no quicksaving capability in the Scenario Editor, just regular save (as...).